### ISSUE TITLE
Log terminal needs to support color output character

### ISSUE THREAD

**Issue by klzsysy** [openshift/console#9126]
If contained in the application log output color, is unable to display characters

![image](https://user-images.githubusercontent.com/3226923/120617914-22e81500-c48d-11eb-84ad-f4c380b56812.png)

```
[90m2021-06-03T16:28:28+08:00[0m [32mINF[0m [1m../build/pkg/apiserver/handlers/pipelinerunHandler.go:339[0m[36m >[0m generate selector: [target-environment=MANAGE-DEV target-environment=ALL] [36mpkg=[0mhandlers
[90m2021-06-03T16:28:28+08:00[0m [33mDBG[0m [1m../build/pkg/apiserver/handlers/pipelinerunHandler.go:357[0m[36m >[0m generate git-ci.yml success [36mgitlabCI=[0mnull [36mpkg=[0mhandlers
[90m2021-06-03T16:28:28+08:00[0m [33mDBG[0m [1m../build/pkg/apiserver/handlers/pipelinerunHandler.go:148[0m[36m >[0m pipelineRun created [36mpipelineRun=[0m{"metadata":{"creationTimestamp":"2021-06-03T08:28:28Z","generateName":"develop-cicd-","generation":1,"labels":{"group":"launcher","parent-pipeline-id":"186620","pipeline-name":"develop-cicd","project":"cicd","root-group":"launcher"},"managedFields":[{"apiVersion":"launcher.forchange.cn/v1beta1","fieldsType":"FieldsV1","fieldsV1":{"f:metadata":{"f:generateName":{},"f:labels":{".":{},"f:group":{},"f:parent-pipeline-id":{},"f:pipeline-name":{},"f:project":{},"f:root-group":{}},"f:ownerReferences":{".":{},"k:{\"uid\":\"7d9cfd8c-ca9c-46ef-bd6f-833c81a8162e\"}":{".":{},"f:apiVersion":{},"f:blockOwnerDeletion":{},"f:controller":{},"f:kind":{},"f:name":{},"f:uid":{}}}},"f:spec":{".":{},"f:description":{},"f:gitLabRepo":{".":{},"f:commitSHA":{},"f:group":{},"f:project":{},"f:ref":{}},"f:pipelineRunHistory":{".":{},"f:timeLimit":{}},"f:stages":{}},"f:s...
[GIN] 2021/06/03 - 16:28:28 | 201 |  157.425259ms |     10.16.23.49 | POST     "/namespaces/launcher/pipelineruns"
[90m2021-06-03T16:28:28+08:00[0m [32mINF[0m [1m../build/pkg/apiserver/handlers/apiserver.go:164[0m[36m >[0m response: HTTP/1.1 201 Created
Connection: close
```

In the old oc3 is normal， But the oc4 does not work

**Comment by spadgett** [comment]
cc @alimobrem

**Comment by spadgett** [comment]
This is being tracked by JIRA issue https://issues.redhat.com/browse/CONSOLE-402. Closing the GitHub issue to avoid tracking it twice. Please follow the JIRA for updates.

**Comment by spadgett** [comment]
This will be addressed by #10486.
